geonode/harvesting/harvesters/wms.py

```diff
@@ -86,13 +92,27 @@ def get_extra_config_schema(cls) -> typing.Optional[typing.Dict]:
         }
 
     def get_num_available_resources(self) -> int:
-        raise NotImplementedError
+        return len(self._get_useful_layers())
 
     def list_resources(
             self,
             offset: typing.Optional[int] = 0
     ) -> typing.List[base.BriefRemoteResource]:
-        raise NotImplementedError
+        # it is just 1 offset
+        if offset != 0:
+            return []
```

Comment:
Please include a comment that explains why the `offset` parameter is being handled like this, as this may be surprising for future GeoNode developers.

Maybe something like this:

```
# look at `tasks.update_harvestable_resources()` in order to understand the purpose of the 
# `offset` parameter. Briefly, we try to retrieve resources in batches and we use `offset` to 
# control the pagination of the remote service. Unfortunately WMS does not really have the 
# concept of pagination and dumps all available layers in a single `GetCapabilities` response.
# With this in mind, we only handle the case where `offset == 0`, which returns all available resources
# and simply return an empty list when `offset != 0` 
```

Reply:
Updated @giohappy